source/fundamentals/crud/query-document.txt

```diff
@@ -0,0 +1,331 @@
+===============
+Specify a Query
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. _query_document_golang:
+
+Overview
+--------
+
+In this guide, you can learn how to specify a query to match a subset
+of documents.
+
+To match a subset of documents, specify a **query filter** containing
+your **match criteria**. Match criteria consists of the fields and
+values you want matched. A query filter contains at least one set
+of match criteria to determine which documents to include in the
+resulting set.
+
+In a query filter, you can match fields with :ref:`literal values
+<go-literal-values>` or with **query operators**. Query
+operators allow you to perform mathematical or logical operations to
+locate documents within a collection.
+
+The following shows the match criteria format with literal values:
+
+.. code-block:: go
+   :copyable: false
+
+   filter := bson.D{{"<field>", "<value>"}}
+
+The following shows the match criteria format with a query operator:
+
+.. code-block:: go
+   :copyable: false
+   
+   filter := bson.D{{"<field>", bson.D{{"<operator>", "<value>"}}}}
+
+This guide explains how to use :ref:`literal values <go-literal-values>`
+and :ref:`query operators <query-operators-go>` with the ``Find()``
+function.
```

Comment:
This language seems somewhat repetitive here. Maybe something like:

"The following examples use :ref:`literal values <go-literal-values>`
and :ref:`query operators <query-operators-go>` with the ``Find()``
function to return a subset of documents."

?